### ISSUE TITLE
[Arch Linux] Impossible to use deepin-music

### ISSUE THREAD

**Issue by funilrys** [linuxdeepin/developer-center#231]
Hello,
I'm not listening to music so often so I can't tell from when the following issue appears ...

So I wanted to hear the **"25th Anniversary Special Orchestra CD"** Of _The Legend of Zelda_ and I discovered a really bad issue :smile_cat: that can be reproduced over 3/3 Deepin installations ...

Tested format: `*.mp3`, `*.wav`

# Screenshot of what I get on screen
![deepinscreenshot20170402182627](https://cloud.githubusercontent.com/assets/14127285/24588955/8208183a-17d2-11e7-9a22-9f231e1d4f4d.png)

# Debug

``` bash
$ deepin-music
"dtkwidget" can not find qm files
"deepin-music" can not find qm files
2017-04-02, 18:25:48.226 [Debug  ] [qsettingbackend.cpp  Dtk::QSettingBackend::QSettingBackend 36] create config "/home/funilrys/.config/deepin/deepin-music/config.ini"
2017-04-02, 18:25:48.241 [Debug  ] [pluginmanager.cpp    PluginManager::loadPlugins          49] load plugin from: "/usr/bin/../lib/deepin-music/plugins"
2017-04-02, 18:25:48.243 [Debug  ] [pluginmanager.cpp    PluginManager::loadPlugins          52] load plugin so file: "/usr/lib/deepin-music/plugins/libnetease-meta-search.so"
2017-04-02, 18:25:48.472 [Debug  ] [pluginmanager.cpp    PluginManager::loadPlugins          57] load plugin instance: NeteaseMetaSearchEngine(0x182d800) 0x182d810
2017-04-02, 18:25:49.478 [Debug  ] [musicapp.cpp         MusicApp::init                      86] TRACE: start prepare data
2017-04-02, 18:25:49.604 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       48] TRACE: database init finished
2017-04-02, 18:25:49.605 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       51] TRACE: player init finished
2017-04-02, 18:25:49.605 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       54] TRACE: Settings init finished
2017-04-02, 18:25:49.606 [Debug  ] [metasearchservice.cpp MetaSearchService::init             76] load meta search plugins
2017-04-02, 18:25:49.606 [Debug  ] [metasearchservice.cpp MetaSearchService::init             80] load meta search engine NeteaseMetaSearchEngine(0x182d800)
2017-04-02, 18:25:49.607 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       59] TRACE: lyricService init finished
2017-04-02, 18:25:49.630 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       64] TRACE: library init finished
2017-04-02, 18:25:49.652 [Debug  ] [presenter.cpp        PresenterPrivate::initBackend       68] TRACE: playlistMgr init finished
2017-04-02, 18:25:49.652 [Debug  ] [presenter.cpp        Presenter::prepareData              141] TRACE: initBackend finished
2017-04-02, 18:25:50.330 [Warning] [                                                         0] MainFrame(0x1847d10, name="MainFrame")  does not have a property named  "borderInside"
2017-04-02, 18:25:50.360 [Debug  ] [musicapp.cpp         MusicApp::onDataPrepared            150] TRACE: data prepared
2017-04-02, 18:25:50.383 [Warning] [                                                         0] QImage::scaled: Image is a null image
2017-04-02, 18:25:50.454 [Debug  ] [footerwidget.cpp     Footer::onModeChange                642] change play mode to 0
2017-04-02, 18:26:05.839 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          232] import form ("/run/user/1000/gvfs/cdda:host=sr0/") to "all"
2017-04-02, 18:26:05.860 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 1.wav"
2017-04-02, 18:26:05.876 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 1.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.877 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 1.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.881 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 2.wav"
2017-04-02, 18:26:05.882 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 2.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.882 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 2.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.885 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 3.wav"
2017-04-02, 18:26:05.886 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 3.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.886 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 3.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.889 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 4.wav"
2017-04-02, 18:26:05.890 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 4.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.890 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 4.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.894 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 5.wav"
2017-04-02, 18:26:05.894 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 5.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.894 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 5.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.898 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 6.wav"
2017-04-02, 18:26:05.899 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 6.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.899 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 6.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.903 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 7.wav"
2017-04-02, 18:26:05.904 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 7.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.904 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 7.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.904 [Warning] [medialibrary.cpp     MediaLibraryPrivate::importMeta     99] skip "*.wav" "/run/user/1000/gvfs/cdda:host=sr0/Track 8.wav"
2017-04-02, 18:26:05.905 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          248] process file "/run/user/1000/gvfs/cdda:host=sr0/Track 8.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.905 [Warning] [medialibrary.cpp     MediaLibrary::importMedias          250] create meta failed: "/run/user/1000/gvfs/cdda:host=sr0/Track 8.wav" QSharedPointer(0x0)
2017-04-02, 18:26:05.905 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          311] 0
2017-04-02, 18:26:05.905 [Debug  ] [medialibrary.cpp     MediaLibrary::importMedias          319] scanFinished "all"
2017-04-02, 18:26:05.906 [Debug  ] [presenter.cpp        Presenter::prepareData::lambda      161] scanFinished
2017-04-02, 18:26:05.906 [Debug  ] [presenter.cpp        Presenter::prepareData::lambda      163] scanFinished: meta library clean
2017-04-02, 18:26:34.483 [Debug  ] [main.cpp             main::lambda                        113] sync config start
2017-04-02, 18:26:35.136 [Debug  ] [main.cpp             main::lambda                        116] sync config finish
2017-04-02, 18:26:35.137 [Debug  ] [musicapp.cpp         MusicApp::~MusicApp                 54] destory MusicApp
2017-04-02, 18:26:35.138 [Debug  ] [musicapp.cpp         MusicApp::~MusicApp                 57] MusicApp destoryed
2017-04-02, 18:26:35.138 [Debug  ] [presenter.cpp        Presenter::~Presenter               124] destory presenter
Segmentation fault (core dumped)
```

**Comment by funilrys** [comment]
Is it just about the meta ? 

https://github.com/linuxdeepin/deepin-music/blob/d5cec57e575540c995a43f666f9e1118ff8f62fb/music-player/core/medialibrary.cpp#L248

**Comment by wucke13** [comment]
I can confirm this, using arch too

**Comment by hualet** [comment]
This should've been fixed with the release of deepin-music 3.0.12.  @funilrys

**Comment by funilrys** [comment]
Tested over 2/3 great job :)
